infra/gcp/terraform/backend.tf

```diff
@@ -72,6 +80,10 @@ resource "google_cloud_run_service" "backend" {
           name  = "NODE_ENV"
           value = "production"
         }
+        env {
+          name  = "AUTH0_DOMAIN"
+          value = "dummy"
+        }
```

Comment:
I would prefer if we fixed this in the source code rather than the infrastructure. IoW, if we're not using Auth0, make it so that the server doesn't require this variable to be set.

Reply:
See [comment below](https://github.com/nearform/titus/pull/795#issuecomment-811709476), need to leave this in place.